packages/node_modules/pouchdb/package.json

```diff
@@ -41,23 +41,6 @@
     "leveldown": "1.4.6",
     "websql": "0.4.4"
   },
-  "devDependencies": {
-    "pouchdb-adapter-fruitdown": "5.5.0-prerelease",
-    "pouchdb-adapter-http": "5.5.0-prerelease",
-    "pouchdb-adapter-indexeddb": "5.5.0-prerelease",
-    "pouchdb-adapter-localstorage": "5.5.0-prerelease",
-    "pouchdb-adapter-memory": "5.5.0-prerelease",
-    "pouchdb-adapter-node-websql": "5.5.0-prerelease",
-    "pouchdb-ajax": "5.5.0-prerelease",
-    "pouchdb-browser": "5.5.0-prerelease",
-    "pouchdb-checkpointer": "5.5.0-prerelease",
-    "pouchdb-core": "5.5.0-prerelease",
-    "pouchdb-generate-replication-id": "5.5.0-prerelease",
-    "pouchdb-node": "5.5.0-prerelease",
-    "pouchdb-promise": "5.5.0-prerelease",
-    "pouchdb-replication": "5.5.0-prerelease",
-    "pouchdb-utils": "5.5.0-prerelease"
-  },
```

Comment:
devDependencies are no longer needed inside of the sub-packages, since everything is handled either at the top-level `node_modules` or in `packages/node_modules`. These devDependencies previously only existed so that `lerna bootstrap` would work correctly.